src/AddressRegistry/StreetName/StreetNameState.cs

```diff
@@ -138,136 +139,151 @@ private void When(AddressWasProposedV2 @event)
+        //    addressToCorrect.Route(@event);
+        //}
+
+        //private void When(AddressPositionWasChanged @event)
+        //{
+        //    var addressToChange = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToChange.Route(@event);
+        //}
+
+        //private void When(AddressPostalCodeWasChangedV2 @event)
+        //{
+        //    var addressToChange = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToChange.Route(@event);
+        //}
+
+        //private void When(AddressPositionWasCorrectedV2 @event)
+        //{
+        //    var addressToCorrect = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToCorrect.Route(@event);
+        //}
+
+        //private void When(AddressPostalCodeWasCorrectedV2 @event)
+        //{
+        //    var addressToCorrect = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToCorrect.Route(@event);
+        //}
+
+        //private void When(AddressHouseNumberWasCorrectedV2 @event)
+        //{
+        //    var addressToCorrect = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToCorrect.Route(@event);
+        //}
+
+        //private void When(AddressBoxNumberWasCorrectedV2 @event)
+        //{
+        //    var addressToCorrect = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToCorrect.Route(@event);
+        //}
+
+        //private void When(AddressWasRemovedV2 @event)
+        //{
+        //    var addressToRemove = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToRemove.Route(@event);
+        //}
+
+        //private void When(AddressWasCorrectedFromRejectedToProposed @event)
+        //{
+        //    var addressToCorrect = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToCorrect.Route(@event);
+        //}
+
+        //private void When(AddressWasRemovedBecauseHouseNumberWasRemoved @event)
+        //{
+        //    var addressToRemove = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToRemove.Route(@event);
+        //}
+
+        //private void When(AddressRegularizationWasCorrected @event)
+        //{
+        //    var addressToRemove = StreetNameAddresses.GetByPersistentLocalId(new AddressPersistentLocalId(@event.AddressPersistentLocalId));
+        //    addressToRemove.Route(@event);
+        //}
+
+        // TODO : voorstel
```

Comment:
proposition

Reply:
I think it's an approvement, but would rename forward to route.